Userland/Services/WindowServer/Screen.cpp

```diff
@@ -23,6 +23,7 @@ NonnullOwnPtrVector<Screen, 8> Screen::s_screens;
 Screen* Screen::s_main_screen { nullptr };
 Gfx::IntRect Screen::s_bounding_screens_rect {};
 ScreenLayout Screen::s_layout;
+Vector<int, 2> Screen::s_scale_factors_in_use;
```

Comment:
Why `2` ? Is it because we support 2 values for the DPI ? Maybe add a comment for this or use the relevant constexpr if there's any :^)